src/mumble/CoreAudio.h

```diff
@@ -6,50 +6,63 @@
 #ifndef MUMBLE_MUMBLE_COREAUDIO_H_
 #	define MUMBLE_MUMBLE_COREAUDIO_H_
 
+#	include <AudioToolbox/AudioToolbox.h>
+
 #	include "AudioInput.h"
 #	include "AudioOutput.h"
 
-#	include <AudioToolbox/AudioToolbox.h>
-#	include <Carbon/Carbon.h>
-
-#	include "Global.h"
+enum AUElement { OUTPUT = 0, INPUT = 1 };
 
 class CoreAudioSystem : public QObject {
 private:
 	Q_OBJECT
 	Q_DISABLE_COPY(CoreAudioSystem)
 public:
-	static CFStringRef QStringToCFString(const QString &str);
-	static const QHash< QString, QString > getDevices(bool input);
+	static const QHash< QString, QString > getDevices(bool input, bool echo);
```

Comment:
How hard would it be to use `std::unordered_map` instead?

Also: missing docs

Reply:
Same concern: Is it good to mix `Q` and `std`?

WIll add docs.

Reply:
I checked and discovered this `QHash` is directly passed to `AudioInputRegistrar`. Refactoring them into `std::unordered_map` will require a rather big refactor. I think I can do this in another PR.